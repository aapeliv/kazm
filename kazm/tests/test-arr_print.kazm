//iterate over array and print

int main() {
  array int[10] my_arr = [1,2,3,4,5,6,7,8,9,10];
  int i = 0;
  int_print(my_arr[0]);
  int_print(my_arr[1]);
  int_print(my_arr[2]);
  int_print(my_arr[3]);
  int_print(my_arr[4]);
  int_print(my_arr[5]);
  int_print(my_arr[6]);
  int_print(my_arr[7]);
  int_print(my_arr[8]);
  int_print(my_arr[9]);
  println("");
  while (i < 10) {
    int_print(i);
    i = i + 1;
  }
  int j = 0;
  println("");
  while (j < 10) {
    int_print(my_arr[j]);
    j = j + 1;
  }
}
