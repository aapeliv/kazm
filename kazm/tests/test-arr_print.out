12345678910
0123456789
12345678910